aeson.cabal

```diff
@@ -110,6 +110,10 @@ library
     -- `Data.Semigroup` is available in base only since GHC 8.0
     build-depends: semigroups >= 0.16.1
 
+  if !impl(ghc >= 7.10)
+    -- `Numeric.Natural` is available in base only since GHC 7.8
+    build-depends: nats>=1
```

Comment:
why no upper bound? Any major version bumps in `nats` from now on will be due to non-backward compatible changes. If there a reason we can leave off the upper bound it should be described in a  comment